README.md

```diff
@@ -2,8 +2,14 @@
 
 phpBB's Skeleton Extension is a tool for extension authors to help speed up and simplify the task of starting a new extension project. It generates sample starter files and directories in a skeleton package that you can use to begin building your extension.
 
+## Requirements
+
+- phpBB version 3.1.4 or later.
+- PHP module `ZipArchive` enabled.
```

Comment:
Well, just being clear. If we're specifying PHP modules, might as well reiterate the PHP version (5.3.3).